docs/interactive-mode.rst

```diff
@@ -33,7 +33,7 @@ The ``generate-tags`` command takes two arguments:
 - ``filename``: name of the file to generate tags for (required)
 - ``size``: size in bytes of the file, if the contents will be received over stdin (optional)
 
-The simplest way to generate tags for a file is by passing its path on disk. The response will include
+The simplest way to generate tags for a file is by passing its path on filesystem(``fileio request``). The response will include
```

Comment:
How about `--_interactive=fileio` ? Do you think `--_interactive=file` is better?
If yet, I will do s/fileio/file/g.